manageiq-gems-pending.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib", "lib/gems/pending", "lib/gems/pending/util"]
 
-  s.required_ruby_version = "> 2.2.2"
+  s.required_ruby_version = "> 2.3.1"
```

Comment:
I wonder if this is should be 2.4?

https://www.ruby-lang.org/en/news/2019/03/31/support-of-ruby-2-3-has-ended/, at least for master...

Reply:
@bdunne @Fryguy what do you think?